analyzer/codechecker_analyzer/analyzers/clangtidy/analyzer.py

```diff
@@ -203,6 +179,28 @@ def get_checker_list(self, config) -> Tuple[List[str], List[str]]:
             elif state == CheckerState.disabled:
                 checkers.append('-' + checker_name)
 
+        # -checks=-clang-analyzer-* option is added to the analyzer command by
+        # default except when all analyzer config options come from .clang-tidy
+        # file. The content of this file overrides every other custom config.
```

Comment:
This comment is no longer valid. Do we want this to be valid, or should we remove this comment?